pallets/common/src/constants.rs

```diff
@@ -43,9 +43,12 @@ pub mod fee {
 /// DID-related.
 pub mod did {
     /// prefix for user dids
-    pub const USER: [u8; 5] = *b"USER:";
+    pub const USER: &[u8; 5] = b"USER:";
     /// prefix for security token dids
-    pub const SECURITY_TOKEN: [u8; 15] = *b"SECURITY_TOKEN:";
+    pub const SECURITY_TOKEN: &[u8; 15] = b"SECURITY_TOKEN:";
+    /// Governan Committee DID. It is used in syste
```

Comment:
minor - looks like comment got chopped - please can we add a comment for the CDD_PROVIDERS_ID as well

Reply:
Fixed.